pkg/listener/kafka/envs

```diff
@@ -0,0 +1,2 @@
+export BROKER_LIST=kafka.kubeless:9092
```

Comment:
I assume this is for debugging?

Reply:
yeah it's only debugging. Those env vars are declared in the configmap object